pyproject.toml

```diff
@@ -44,30 +44,6 @@ no_implicit_optional = true
 module = ["tests.*"]
 strict = false
 
-[tool.pylint]
-load-plugins = "pylint_fixme_info"
-disable = "fixme"
-
-[tool.pylint.messages_control]
-disable = "too-many-ancestors,too-few-public-methods,fixme,unspecified-encoding,use-implicit-booleaness-not-comparison,unnecessary-lambda-assignment"
-
-[tool.pylint.similarities]
-min-similarity-lines=13
-
-[tool.pylint.format]
-max-line-length = "88"
-max-attributes = 16
-max-args= 6
-max-locals = 18
-
-[tool.pylint.MASTER]
-extension-pkg-whitelist = [
-    "apt_pkg",
-    "pydantic"
-]
-load-plugins = "pylint_fixme_info,pylint_pytest"
-good-names = "id"
-
 [tool.ruff]
```

Comment:
Could you update this section to match the latest starbase? Some of the linter rules we had enabled in pylint (e.g. unused arguments) aren't enabled here.